vyper/compiler/phases.py

```diff
@@ -174,7 +175,11 @@ def resolved_imports(self):
     def _annotate(self) -> tuple[natspec.NatspecOutput, vy_ast.Module]:
         module = self._resolve_imports[0]
         analyze_module(module)
-        nspec = natspec.parse_natspec(module)
+        try:
+            nspec = natspec.parse_natspec(module)
```

Comment:
nvm -- NatspecSyntaxException inherits from SyntaxException. however it will be better to have the try/catch in `parse_natspec` itself (in case it's ever used from somewhere else)